azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java

```diff
@@ -46,6 +54,19 @@ protected void doExecute() throws MojoExecutionException, MojoFailureException {
         getLog().info(deploymentClient.getDeploymentStatus().toString());
     }
 
+    protected boolean checkProjectPackaging(MavenProject project) throws MojoExecutionException {
```

Comment:
protected static boolean isPomPackingProject(MavenProject proj) {
        return POM.equals(proj.getPackaging());
    }

     protected static boolean isJarPackingProject(MavenProject proj) {
        return JAR.equals(proj.getPackaging());
    }

Reply:
add and use these two methods

Reply:
Done